### PR TITLE
Fix: misspelled word

### PR DESCRIPTION
**Pull request by HamedMasafi**

### CODE REVIEW

**Comment by EndrII** [comment]
@HamedMasafi thanks you, hm tests is failed and i see empty commit 1d1d36de5d9650bf6a10d42c00f43f4515030eff
please squash your changes to one commit.

File: src/Deploy/src/configparser.cpp [hunk 1]
```diff
@@ -708,7 +708,7 @@ bool ConfigParser::parseDeployMode(bool checkBin) {
         QuasarAppUtils::Params::log("Failed to initialize targets or extra data!",
                                     QuasarAppUtils::Error);
 
-        QuasarAppUtils::Params::log("Use bin or extraData optins. And check input pathes.",
+        QuasarAppUtils::Params::log("Use bin or extraData options. And check input pathes.",
```
Comment (pzhlkj6612) [inline src/Deploy/src/configparser.cpp:711]:
```suggestion
        QuasarAppUtils::Params::log("Use bin or extraData options and check input paths.",
```

https://www.dictionary.com/browse/path

**Review by pzhlkj6612 — COMMENTED** [review]
Let's make it better.

**Comment by EndrII** [comment]
@HamedMasafi oh looks as something broken on CI, I will merge this Pr, after fix Ci issues.

**Comment by EndrII** [comment]
@HamedMasafi thanks you )

**Pull request merged**